app/src/main/java/com/github/neighbortrader/foodboardapp/MainActivity.java

```diff
@@ -61,23 +61,69 @@ protected void onCreate(Bundle savedInstanceState) {
 
         TextView textView = findViewById(R.id.textView2);
         textView.setKeyListener(null);
+        textView.setMovementMethod(new ScrollingMovementMethod());
+
+        fabPostOffers.setOnClickListener(view -> {
+            Snackbar.make(view, "Create new random offer", Snackbar.LENGTH_SHORT)
+                    .setAction("Action", null).show();
+
+            textView.scrollTo(0, 0);
+
+            StringBuffer editTextWithAllReceivedOffers = new StringBuffer();
+
+            OfferHandler.builder(RequestTyps.POST_NEW_OFFER, getApplicationContext(), new OnEventListener<OfferHandler>() {
+                @Override
+                public void onResponse(OfferHandler offerHandler) {
+
+                    Offer offer = offerHandler.getOfferToPost();
+
+                    editTextWithAllReceivedOffers.append("Successfully added Offer\n\n");
+
+                    editTextWithAllReceivedOffers.append("Beschreibung: " + offer.getDescription()).append("\n")
+                            .append("Preis: " + offer.getPrice().getFormattedPrice())
```

Comment:
Wie wäre es den Code hier auszulagern Handler bspw

Reply:
Grundsätzlich ja und das ist auch geplant, jedoch ist die MainActivity aktuell in diesem Testzustand. Sobald wir anfangen die UI anzubinden werden entsprechende Händler und Funktionen erstellt.

Reply:
CleanCode von Anfang an schreiben ... sonst rodden Code ;)

Reply:
Schwatz nicht review